lib/ASRouter.jsm

```diff
@@ -289,6 +320,38 @@ const MessageLoaderUtils = {
             provider.id,
             options.dispatchToAS
           );
+        } else if (["cfr", "cfr-fxa"].includes(provider.id)) {
+          // Both "cfr" and "cfr-fxa" require the Fluent file for l10n, currently
+          // they share the same attachment of the record "cfr-${version}-${locale}",
+          // e.g. for "en-US" with version "v1", the record ID is "cfr-v1-en-US".
+
+          const locale = Services.locale.appLocaleAsLangTag;
+          const recordId = `cfr-${RS_FLUENT_VERSION}-${locale}`;
```

Comment:
Is it because cfr and cfr-fxa share the same record id that it's not a module constant?

Reply:
Yep, I think we can make a constant for that.